frontend_tests/puppeteer_tests/03-compose.ts

```diff
@@ -59,9 +59,9 @@ async function test_keyboard_shortcuts(page: Page): Promise<void> {
 }
 
 async function test_reply_by_click_prepopulates_stream_topic_names(page: Page): Promise<void> {
-    const stream_message = get_last_element(
-        await page.$x(get_message_xpath("Compose stream reply test")),
-    );
+    const stream_message_xpath = get_message_xpath("Compose stream reply test");
+    await page.waitForXPath(stream_message_xpath, {visible: true});
```

Comment:
@andersk should this one by `hidden: false` instead?

Reply:
No, both `visible: false` and `hidden: false` are default, so setting either to `false` again would never be useful. If you want one of them you have to set one of them to `true`.

(I haven’t checked whether we need the option here at all, though.)

Reply:
@andersk This is just a precaution from a rare flake I noticed [here](https://app.circleci.com/pipelines/github/Riken-Shah/zulip/145/workflows/b9652d26-18a8-40ff-9d90-6929c54b46b0/jobs/654) when I ran `03-compose.ts` for 100x in Firefox mode. 
I thought it failed because compose box closing and stream message click happened at the almost same time. So after adding these changes I ran it again and it ran successfully ([link to the workflow](https://app.circleci.com/pipelines/github/Riken-Shah/zulip/159/workflows/b543ba62-b1d7-4218-908e-cc90ebe5cb70/jobs/686)).